paths.tex

```diff
@@ -77,9 +77,9 @@ \section{Preliminary Definitions}
 directed temporal networks. To go from undirected to directed, for every edge
 $(e_l,e_r) \in E^2$, add edges $e_l \in E$ and $e_r \in E$ to the directed
 network. To go from directed to undirected, if for every $u,v$ pair in the
-directed network, the number of edges of the form $(u,v,t_0,t_f,w)$ is the same
-as the number of edges of the form $(v,u,t_0,t_f,w)$, then pair up these edges;
-otherwise there is no transformation.
+directed network, the number of edges of the form $e = (u,v)$ is the same
```

Comment:
Looking over this again, I'm not sure the transformation stated like this makes sense since the the graph is a multi-graph; given nodes *a* and *b*, there could exist *a*->(1990)*b* and *b*->(1990)*a* such that these two edges were not generated from the same entity (i.e. this is a contextual definition). What are your thoughts? @ericthewry